### PR TITLE
KFSPTS-22704 Reintroduce KFSPTS-13005 billing period fix

### PR DESCRIPTION
**Pull request by jhulslander**

### CODE REVIEW

File: src/test/java/org/kuali/kfs/module/ar/businessobject/TimeBasedBillingPeriodTest.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+    @Test
+    public void testDetermineStartDateByFrequencyBasic() {
+        Date lastBilledDate = buildDate(2018, Calendar.NOVEMBER, 1);
+        Date expectedNextDate = buildDate(2018, Calendar.NOVEMBER, 2);
+        assertEquals(expectedNextDate.toString(), findNextBillingDate(lastBilledDate).toString());
+    }
+
+    @Test
+    public void testDetermineStartDateByFrequencyNewYeara() {
```
Comment (cah292) [inline src/test/java/org/kuali/kfs/module/ar/businessobject/TimeBasedBillingPeriodTest.java:48]: Minor typo in method name.

**Review by cah292 — CHANGES_REQUESTED** [review]
I found one very minor item that would be good to clean up. Other than that, the changes look good and they appear to be reintroducing the proper KFSPTS-13005 functionality.

**Comment by jhulslander** [comment]
this is ready for review again

**Review by cah292 — APPROVED** [review]
Changes look good, and unit tests pass. Merging.

**Pull request merged**